File: sample/src/personaclick/java/com/personaclick/sample/SampleApplication.java
package com.personaclick.sample;

import com.personaclick.sdk.Personaclick;
import com.personalizatio.sample.AbstractSampleApplication;

/**
 * Created by Sergey Odintsov
 *
 * @author nixx.dj@gmail.com
 */
public class SampleApplication extends AbstractSampleApplication<Personaclick> {

	@Override
	protected String getShopId() {
		return "cb0516af5da25b1b41490072e679bc";
	}

	@Override
	protected void initialize() {
		Personaclick.initialize(getApplicationContext(), getShopId());
	}
}

File: sample/src/personaclick/kotlin/com/personaclick/sample/SampleApplication.kt
package com.personaclick.sample

import com.personaclick.sdk.Personaclick
import com.personalization.sample.AbstractSampleApplication

class SampleApplication : AbstractSampleApplication<Personaclick>(Personaclick.getInstance()) {
    override val shopId: String
        get() = "cb0516af5da25b1b41490072e679bc"

    override fun initialize() {
        Personaclick.initialize(
            context = applicationContext,
            shopId = shopId,
        )
    }
}
